### DISCUSSION TITLE
Path to screen mirroring app

### DISCUSSION THREAD

**Discussion by wonder99** [springbok/MLM2PRO-GSPro-Connector#37] [General]
For android, the best mirroring app is scrcpy, and it's normally required to run adb before running scrcpy.
I use a batch file to do that, and also provide command-line arguments to scrcpy, so it would be good to support batch files as well as arguments

**Comment by springbok** [comment]
Ok will do.

**Reply by springbok** [reply]
I've added .bat files to allowed file types but not sure how you want the arguments to work, did you want the connector to pass these arguments to the batch file? Can you give me an example?

**Comment by wonder99** [comment]
Well, if .bat files are supported, then you may not really need the
arguments.  It's just that some screen mirror programs might work better
with some arguments (like scrcpy can be told to turn the device screen off,
or maximize the mirrored window immediately with arguments).
For me, I need .bat to make sure the 'adb' is running before scrcpy, to run
over wifi rather than USB.  Because of that, I need .bat anyway, and
arguments can just be included in the .bat.
I just don't know how easy you want to make this--do you want people to
have to write a .bat file to include arguments?  It's OK by me.
- Paul
